README.md

```diff
@@ -6,9 +6,27 @@
 
 Build a React app containing:
 
-- An input where the user can enter the values in an array
+- An input where the user can enter comma-separated number values in an array
 - A select to choose between calculating the sum, the average, and the [mode](https://www.mathsisfun.com/definitions/mode.html) (most frequently occurring number)
-- A calculate button
+- A calculate button that, when pressed, puts the result of the selected operation on the input numbers on the screen
+  - If the numbers are invalid for some reason, put `Invalid input.`
+
+For example, if the text input contains `1,3,3,7` and the select is set to `sum`, clicking the button should print `14` to the screen.
+Switching the select to `mode` and clicking the button again would replace that with `3`.
+
+![reactFormsLab](./reactFormsLab.gif)
+
+| input | operation | result         |
+| ----- | --------- | -------------- |
+| 1     | average   | 1              |
+| 1     | mode      | 1              |
+| 1     | sum       | 1              |
+| 1,3,5 | average   | 2              |
+| 1,3,5 | mode      | 3              |
+| 1,3,5 | sum       | 9              |
+|       | \*        | 1              |
```

Comment:
If there's no input or operation, the result is `1`? I am not a math person, is that the expected result?

Reply:
Lol sometimes your kind comments are... too kind. Just a copypasta oversight.